core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -43,8 +43,9 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
       <div class="title-section-mask"></div>
 
       <div ng-attr-section="<[isWindowLarge ? undefined : 'right-section']>">
-        <span ng-if="getObjective()" class="oppia-activity-summary-tooltip">
-          <span tooltip="<[getObjective()]>" tooltip-placement="top">
+        <span ng-if="getObjective()" class="oppia-activity-summary-popover">
+          <span popover="<[getObjective()| truncateAndCapitalize]>" popover-placement="top" 
+          popover-trigger="mouseenter">
```

Comment:
Nit: Indent so that left edge of 'popover-trigger' matches left edge of 'popover'